README.rst

```diff
@@ -94,7 +94,7 @@ To run framework tests with ``pytest``::
     pip install pytest
     ANYBLOK_DATABASE_DRIVER=postgresql ANYBLOK_DATABASE_NAME=test_anyblok py.test anyblok/tests
 
-To run tests of all installed bloks::
+To run tests of all installed bloks with demo data::
 
     anyblok_createdb --db-name test_anyblok --db-driver-name postgresql --install-all-bloks
```

Comment:
missing option --with-demo

Reply:
In fact I removed it for anyblok tests as  the [test to validate skip_unless_demo_data_installed](https://github.com/AnyBlok/AnyBlok/pull/151/files#diff-a5538a98168795af2ac1812c16b6ed6dR953) assume database to be created without `--with-demo`

Reply:
I've moved that test here https://github.com/AnyBlok/AnyBlok/pull/151/files#diff-065233cad8ea7b9d2e4219e4f3c972fdR1

As using this pytest fixture `rollback_registry` in `anyblok/tests` directory had some side effect.

@jssuzanne I could probably implement a way to skip in case demo data are installed ? what do you think ?
* skip_unless_demo_data_installed
* skip_while_demo_data_installed

In that case I would test both and read the `ẁith_demo` System.parameter value, only one test should be executed according i we are using --with-demo while setup anyblok test database.

Reply:
done by https://github.com/AnyBlok/AnyBlok/pull/151/commits/83178ea5ceaae6b576054258c28b9a82adb6c437